coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -32,14 +34,20 @@ def __init__(self, documentation, docstyle_definition,
         :param marker:
             The three-element tuple with marker strings, that identified this
             documentation comment.
-        :param range:
-            The position range of type ``TextRange``.
+        :param position:
+            The starting ``TextPosition`` of the documentation.
         """
         self.documentation = documentation
         self.docstyle_definition = docstyle_definition
         self.indent = indent
         self.marker = marker
-        self.range = range
+        self.position = position
+        self.assemble = self.assemble()
```

Comment:
@Makman2 do you want me to remove `self.assemble` ? and use `self.assemble()` instead ?